cips/README.md

```diff
@@ -20,6 +20,7 @@ Read [CIP-1](./cip-1.md) for information on the CIP process.
 | 12 |   June 5, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/149) |    [Tweet Thread](https://x.com/JoshCStein/status/1798381092493504830)    |    [Recording](https://www.youtube.com/watch?v=kV5_ANNhixI)     |
 | 13 |   July 3, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/155) |    [Tweet Thread](https://x.com/JoshCStein/status/1808531780506403102)    |    [Recording](https://youtu.be/WzPi49Ge2rA)     |
 | 14 |   July 24, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/163) |    [Tweet Thread](https://x.com/JoshCStein/status/1816142425011662993), [Notes](./notes/cdc-14.md)    |    [Recording](https://www.youtube.com/watch?v=bwlpAy_KE9U)     |
+| 15 |   August 7, 2024    | [Agenda](https://github.com/celestiaorg/CIPs/issues/185) |    N/A    |    N/A    |
```

Comment:
```suggestion
| 15 |   August 7, 2024 (Scheduled)   | [Agenda](https://github.com/celestiaorg/CIPs/issues/185) |    Tweet Thread, Notes    |    Recording    |
```

Reply:
I'm fine with what you previously had or this change.